Engine/Localization.h

```diff
@@ -3,347 +3,417 @@
 #include "Engine/Objects/NPCProf.h"
 
 
-#define LSTR_AC                          0   // "AC"
```

Comment:
had to add extra spaces and it's now all marked as changed...
the actual change is adding a lot of LSTR_FMT_xxx